js/packages/teams-ai/src/AI.ts

```diff
@@ -246,6 +246,13 @@ export class AI<TState extends TurnState = TurnState> {
         return this._options.planner;
     }
 
+    /**
+     * @returns {boolean} Returns the feedback loop flag.
+     */
+    public get enableFeedbackLoop(): boolean {
```

Comment:
Is this needed for streaming?

Reply:
Yup, it needs to be passed into the StreamingResponse class